mlflow/metrics/__init__.py

```diff
@@ -263,6 +264,32 @@ def rougeLsum() -> EvaluationMetric:
     )
 
 
+precision_at_k = make_metric(
+    eval_fn=_precision_at_k_eval_fn,
+    greater_is_better=True,
+    name="precision_at_k",
+    version="v1",
+)
```

Comment:
can we use the factory function approach here? see `mae()` below as an example. users should get an EvaluationMetric object by calling `mlflow.metrics.precision_at_k()`

Reply:
Sure, will change.